docs/watching-assets.mdx

```diff
@@ -6,43 +6,41 @@ Valora includes a default list of popular tokens. Each token in the
 default list must meet the asset listing <a href={props.assetGuidelines}>requirements</a>.
 However, there are many tokens that users
 want to manage and transact with in Valora that do not meet the requirements. To
-support these tokens well, Valora implements
-[EIP-747](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-747.md),
-which allows dapps to make it easy for users to add arbitrary tokens
-to Valora.
+support these tokens well, Valora implements a deeplink handler so
+that a dapp can construct deeplinks that make it easy for their users
+to add arbitrary tokens to Valora.
 
-## Example
+## Using deeplinks
+
+> This feature is in-development and not fully supported yet!
+
+Valora's deeplink handler requires the contract address. If your
+ERC-20 contract supports
+[EIP-1046](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1046.md),
+Valora will fetch the token symbol, decimals, and image from the token
+metadata. If your ERC-20 contract does not support EIP-1046 you must
+specify symbol, decimals, and optionally image in the deeplink. You
+can also override the image specified in an EIP-1046 compliant
+contract.
```

Comment:
`symbol` and `decimals` are already optional in the [ERC-20 spec](https://eips.ethereum.org/EIPS/eip-20#methods).

Not sure how to word this though:

```suggestion
Valora's deeplink handler requires the contract address. If your
ERC-20 contract supports
[EIP-1046](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1046.md),
Valora will fetch the token symbol, decimals, and image from the token
metadata. If your ERC-20 contract does not support EIP-1046, but still implements the `symbol` and `decimals` methods, it will use them. Otherwise you must
specify symbol, decimals, and optionally image in the deeplink. You
can also override the image specified in an EIP-1046 compliant
contract.
```